### PR TITLE
adding postal codes for more countries, now its capable to support mo…

### PR DESCRIPTION
**Pull request by neerajkumar**

### CODE REVIEW

File: lib/validates_zipcode/cldr_regex_collection.rb [hunk 1]
```diff
@@ -162,7 +162,107 @@ module CldrRegexpCollection
       TC: /\ATKCA 1ZZ\z/,
       WF: /\A986\d{2}\z/,
       XK: /\A\d{5}\z/,
-      YT: /\A976\d{2}\z/
+      YT: /\A976\d{2}\z/,
+      AE: /\A([a-zA-Z\d\s]){3,}\z/,
```
Comment (dgilperez) [inline lib/validates_zipcode/cldr_regex_collection.rb]: Hey @neerajkumar ! Thanks for the PR. I'm just having a look at the new additions and I have a few questions: 

* What are your sources for the new formats? I'd like to be sure we add only valid regexs.
* I see a lot of `{3,}` in the new regexs: these zipcodes have only a minimum length, but no maximum. Is that correct for all those countries, or is it just an approximation?

Thanks again!

File: Rakefile [hunk 2]
```diff
@@ -3,3 +3,8 @@ require 'rspec/core/rake_task'
 
 RSpec::Core::RakeTask.new(:spec)
 task default: :spec
+
+desc "Open an irb session preloaded with this library"
```
Comment (dgilperez) [inline Rakefile]: Just curious: what do you use this for?

File: lib/validates_zipcode/cldr_regex_collection.rb (hunk 1 above)
Comment (neerajkumar) [inline lib/validates_zipcode/cldr_regex_collection.rb]: Actually in one of my project, I used this gem and got some more requirements for other countries too. So I manually created the regex based on its respective format. As per my knowledge its correct format and we can go ahead and add.

File: Rakefile (hunk 2 above)
Comment (neerajkumar) [inline Rakefile]: Its for creating an irb session which will load the gem so that we can test for its methods and datas.

**Comment by dgilperez** [comment]
Thanks a lot!

**Comment by dgilperez** [comment]
Could you please update the Readme as well? Now there are more than 159 countries :+1:

**Comment by neerajkumar** [comment]
updated.

On 7 July 2015 at 15:50, David Gil <notifications@github.com> wrote:

> Could you please update the Readme as well? Now there are more than 159
> countries [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dgilperez/validates_zipcode/pull/2#issuecomment-119160952>
> .
>



-- 

*With Regards,**Neeraj Kumar*

**Pull request merged**